encryption_works.md

```diff
@@ -96,6 +96,77 @@ It's important to remember that just because you use free software doesn't mean
+
+Journalists' e-mail accounts are often the subject of attacks, but it's more common for hackers to try and trick Gmail into resetting the password rather than hacking Google itself. There's even a phrase for this -- *rubber-hose cryptanalysis* refers to the fact that it's far easier to coerce or trick someone into giving you their password than break the underlying encryption.
+
+!["Security." Courtesy of XKCD.](http://imgs.xkcd.com/comics/security.png)
+
+Let's look at another example: a metro journalist reporting on corruption in her city's police force. This is a story which will have sensitive documents to store, and sources to protect. These are the assets we're trying to protect, so that the corrupt officials in question aren't tipped off in advance about the story. 
+
+The adversity in this case isn't the NSA, but the scenario shows why practicing good operational security protects you from numerous adversities. Snowden is the most famous recent example of a whistleblower, but journalists use inside sources who need protecting quite often.
+
+### Strong Passphrases
+
+The first meaningful action you can take towards ensuring your digital security is to protect your computer and online accounts with a strong password. This is the first —and most important— line of defense between you and anyone with physical access to your computer who wants to impersonate you. 
+
+Traditionally, computer users were encouraged to create and choose a password and attach some numbers onto the end of it. This would lead to passwords such as "Tr0ub4dor&3," to reference a popular [XKCD comic](https://xkcd.com/936/). The problem with this password is that it's hard to remember and it's not especially secure against a brute-force password-guessing attack.  
+
+Passphrases are longer than passwords."The quick fox jumps over the lazy dog" is a long and memorable passphrase, but is a common phrase, so it's not a good idea to use it. The aforementioned XKCD comic's advice is to come up with a passphrase of unconnected words, such as "correct horse battery staple." This is good advice, but it's surprisingly hard to come up with four or five words that are truly unconnected. Humans are bad at randomness, and words often come in groups -- I say brown, you think cow; I say garden, you think shed.
+
+The solution is to externally randomize the words that constitute your passphrase. This is where [Diceware](http://world.std.com/~reinhold/diceware.html) comes in -- using dice to choose the words of your passphrase increases its randomness and its security.
+
+Additionally, Freedom of the Press Foundation recommends using a password manager such as [KeePassX](https://www.keepassx.org). The app is unlocked by a master passphrase, which you memorize.
```

Comment:
I think we can go further to talk about the key components of good password security, as well as in providing a specific set of recommendations.

Key components are:

1. Passphrases should be strong (already covered)
2. Passphrases should be unique (attacks based on reuse are incredibly common)
3. Passphrases should be available - otherwise people will revert to using weak non-unique passwords for convenience.

As for the specific set of recommendations, I would say:

1. Use a password manager like KeePassX (or 1Password, LastPass, etc.)
2. Generate strong, unique passphrases for every site and account you need
3. Develop a way to make those passwords available so you don't revert to or continue to use weak, repeated passphrases. Many password management apps have sync features and mobile apps. Depending on your threat model, you may not want to entrust your password database to a 3rd party, so that should be mentioned as a potential consideration. Another option is to store an encrypted password database file (like a KeePassX `.kdb`) on a synced cloud storage service such as Dropbox, although that can require some forethought (e.g. race conditions writing to the db across devices need to be intentionally avoided).